crates/swc_html_minifier/src/option.rs

```diff
@@ -44,6 +44,20 @@ pub enum CollapseWhitespaces {
     OnlyMetadata,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "kebab-case")]
+pub enum RemoveRedundantAttributes {
```

Comment:
Let's implement `Default` for this type instead of using `#[serde(default = "")]`.

Reply:
You can then change `#[serde(default = "...")]` to `#[serde(default)]`